.github/workflows/build_branch.yaml

```diff
@@ -0,0 +1,35 @@
+name: Build-Deploy-Npm-Canary
+on:
+  push:
+    branches-ignore:
+      - 'master'
+    paths-ignore:
+      - package.json
+
+jobs:
+  build-and-push-to-npm:
+    name: Bygg og push
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - name: Prepare repository
+        run: git checkout "${GITHUB_REF:11}"
```

Comment:
hva gjør denne egentlig?

Reply:
Magi for at man skal få lov til å pushe tilbake til git. Skal vist ha blitt fikset med v2 av checkout, men da må man løse et annet problem: nemlig at man kun får siste commit og da skjønner ikke lerna noen ting av historikken :)